reinforcement_learning/examples/python/basic_usage.py

```diff
@@ -16,23 +16,31 @@ def main():
     model = rl_client.live_model(config, test_cb)
     model.init()
 
-    event_id = "event_id"
+    uuid = "event_id"
     context = '{"User":{"id":"a","major":"eng","hobby":"hiking"},"_multi":[{"N1":{"F1":"V1"},"N2":{"F2":"V2"}},{"N3":{"F1":"V3"}}]}'
 
-    model_id, chosen_action_id, actions_probabilities = model.choose_rank(event_id, context)
+    model_id, chosen_action_id, actions_probabilities = model.choose_rank(context, event_id=uuid)
 
-    print("event_id: " + event_id)
+    print("event_id: " + uuid)
     print("model_id: " + model_id)
     print("chosen action id: " + str(chosen_action_id))
     print("all action probabilities " + str(actions_probabilities))
 
-    model_id, chosen_action_id, actions_probabilities, event_id = model.choose_rank(context, deferred = True)
+    model_id, chosen_action_id, actions_probabilities, event_id = model.choose_rank(context, deferred = False)
+    model.report_action_taken(event_id)
+    print("event_id: " + event_id)
+    print("model_id: " + model_id)
+    print("chosen action id: " + str(chosen_action_id))
+    print("actions probabilities list: " + str(actions_probabilities))
+
+    model_id, chosen_action_id, actions_probabilities = model.choose_rank(context, deferred=True, event_id='asd')
```

Comment:
I agree with Yann and Tyler. I would do:
import uuid
uuid = str(uuid.uuid4())
model_id, chosen_action_id, actions_probabilities = model.choose_rank(context, event_id=uuid, deferred=True)

Reply:
Actually, i thought we said that the deferred action logic will not be exposed except for advanced users.
Maybe we can omit to give an example where deferred is true?

Reply:
That's a good point actually. @ataymano what do you think?

Reply:
are we exposing example? Even if we do, it is secondary thing compare to documentation in my opinion. Should I also remove it from docs?

Reply:
Neither one nor the other IMO